src/Files.App/Converters/UInt32ToString.cs

```diff
@@ -7,12 +7,7 @@ internal class UInt32ToString : IValueConverter
 	{
 		public object Convert(object value, Type targetType, object parameter, string language)
 		{
-			if (value is not null)
-			{
-				return value.ToString();
-			}
-
-			return "";
+			return value is not null ? value.ToString() : "";
```

Comment:
I think you should change `""` with `string.Empty`

Reply:
Done.